js/build.js

```diff
@@ -0,0 +1,344 @@
+Fliplet.Widget.instance({
+  name: 'file-list',
+  displayName: 'File list',
+  icon: 'fa-exchange',
+  render: {
+    dependencies: [],
+    template: [
+      '<div data-view="content"></div>',
+    ].join(''),
+    ready: async function () {
+      var thisy = this;
+      const populateFileList = (entryID = Fliplet.Navigate.query.dataSourceEntryId, dsID = thisy.fields.dataSource) => {
+        var dataSourceId = dsID;
+        var dataSourceEntryId = entryID;
```

Comment:
Inconsistent use of camelCase style

```suggestion
        var dataSourceEntryId = entryId;
```